keras_retinanet/models/densenet.py

```diff
@@ -0,0 +1,77 @@
+"""
+Copyright 2018 vidosits (https://github.com/vidosits/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import keras
+from keras.applications.densenet import DenseNet, get_file
+
+from ..models import retinanet
+
+BASE_WEIGHT_PATH = 'https://github.com/fchollet/deep-learning-models/releases/download/v0.8/'
+
+custom_objects = retinanet.custom_objects.copy()
+
+allowed_backbones = {'densenet121': [6, 12, 24, 16], 'densenet169': [6, 12, 32, 32], 'densenet201': [6, 12, 48, 32]}
+
+
+def download_imagenet(backbone):
+    """ Download pre-trained weights for the specified backbone name. This name is in the format
+        {backbone}_weights_tf_dim_ordering_tf_kernels_notop where backbone is the densenet + number of layers (e.g. densenet121).
+        For more info check the explanation from the keras densenet script itself
```

Comment:
Maybe it'd be nice to add a link to that script here.

Reply:
added.